benchmarks/benchmark_serving.py

```diff
@@ -735,7 +731,8 @@ def main(args: argparse.Namespace):
         "--logprobs",
         type=int,
         default=None,
-        help="Number of logprobs-per-token to return as part of the request.",
+        help=("Number of logprobs-per-token to return as part of the request "
+              "(default 1)"),
```

Comment:
The `(default 1)` is a bit confusing as the default (without specifying this argument) is `None`. How the default 1 applied when I specified `--logprobs`? Or can we just let the default be `None` and always require an integer value when specifying logprobs?

Reply:
Per our discussion, the help doc comment is now "Number of logprobs-per-token to compute & return as part of the request. If unspecified, then either (1) if beam search is disabled, no logprobs are computed & a single dummy logprob is returned for each token; or (2) if beam search is enabled 1 logprob per token is computed"